vm/method-verifier.cpp

```diff
@@ -531,11 +531,19 @@ MethodVerifier::verifyJumpOffset(cell_t offset)
 }
 
 bool
+MethodVerifier::verifyDimensionCount(cell_t ndims)
+{
+    if (ndims <= 0 || ndims > sDIMEN_MAX) {
+      reportError(SP_ERROR_INSTRUCTION_PARAM);
+      return false;
+    }
+    return true;
```

Comment:
Indentation in this function should be 2-space instead of 4. Good to merge w/ that fixed.